torch/fx/node.py

```diff
@@ -78,6 +78,7 @@
 _side_effectful_functions: Set[Callable] = {
     torch._assert,
     torch._assert_async,
+    _ops.aten.copy_.default,
```

Comment:
you shouldn't need to do this. Specifically, this operator should have a schema that says it's mutable and so should automatically be considered side effectful.